docs/data/reads.md

```diff
@@ -0,0 +1,39 @@
+---
+id: reads
+title: Sequencing Reads
+sidebar_label: Sequencing Reads
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis-workflows/analysis-overview). The following page lists genomic read analyses and data files generated by the ARGO Analysis pipelines.
+
+## Submitted Reads
+
+Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline. Data is accepted in FASTQ, unaligned BAM, or aligned BAM formats, and must be compressed for submission.
+
+### File Types
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+## Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format. ARGO reads are aligned to the GRCh38 Human Reference Genome.
```

Comment:
Do we have a link to the GRCh38 Human Reference Genome for download?

Reply:
Its linked here: https://docs.icgc-argo.org/docs/analysis-workflows/dna-pipeline#inputs 

I can also add here if desired.

Reply:
added